docs/examples.md

```diff
@@ -372,6 +392,29 @@ systemd:
         [Install]
         WantedBy=multi-user.target
 ```
+
+This example is uses a file reference to embed a locally defined file (`proxy.conf`) multiple times into the resulting Ignition file during transpile time. This requires using `butane` with the `--files-dir` parameter to allow loading files from disk.
+
+<!-- butane-config -->
+```yaml
+variant: fcos
+version: 1.5.0
+systemd:
+  units:
+    - name: rpm-ostreed.service
+      dropins:
+        - name: 99-proxy.conf
+          contents_local: proxy.conf
+    - name: zincati.service
+      dropins:
+        - name: 99-proxy.conf
+          contents_local: proxy.conf
+    - name: rpm-ostree-countme.service
+      dropins:
+        - name: 99-proxy.conf
+          contents_local: proxy.conf
```

Comment:
Again, this is fairly redundant.  Let's embed one service and one dropin.

Reply:
Yeah, I get what you mean. My intention here was to give an example that immediately shows one of the reasons why one would use file embeds instead of just inlining a systemd service/dropin.